packages/itwinui-react/src/core/Table/TableCell.tsx

```diff
@@ -101,14 +102,18 @@ export const TableCell = <T extends Record<string, unknown>>(
 
   return (
     <>
-      {cell.column.cellRenderer ? (
-        cell.column.cellRenderer({
-          ...cellRendererProps,
-          isDisabled: () => isDisabled,
-        })
-      ) : (
-        <DefaultCell {...cellRendererProps} isDisabled={() => isDisabled} />
-      )}
+      <DefaultCellRendererPropsChildren.Provider
+        value={cellRendererProps.children}
```

Comment:
This should be memoized as it's being passed through Context.

We should also verify how often this value changes in practice.

Reply:
Yes, looks good.